resources/views/layouts/nav.blade.php

```diff
@@ -33,6 +33,7 @@
 
                         <li><a href="/threads?popular=1">Popular Threads</a></li>
                         <li><a href="/threads?unanswered=1">Unanswered Threads</a></li>
+                        <li><a href="/threads?unanswered=1">Active Threads</a></li>
```

Comment:
Looks like a typo here. URL should be `/threads?active=1`.

Reply:
Whoops! Thank you for the hint. I have it correctly in my private iteration of the project and oversaw changing it here instead of just copying the line above.
fixed.